src/components/input/input.test.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import Input from './Input';
+
+describe('test Input', () => {
+
+  it('check render component with title', () => {
+    const component = shallow(<Input
+      value='testValue'
+      name='testName'
+      title='testTitle'
+    />);
+    expect(component.text()).toEqual('testTitle');
+
+    expect(component).toMatchSnapshot();
```

Comment:
❗️
Давай пройдёмся по коду и подровняем всё: консты к констам, экспекты к экспектам. Должно как-то так получиться
```suggestion
    const component = shallow(<Input
      value='testValue'
      name='testName'
      title='testTitle'
    />);
    const whatever = 1;
    
    expect(component.text()).toEqual('testTitle');
    expect(component).toMatchSnapshot();
```

Reply:
fixed